### PR TITLE
helper functions

### PR DESCRIPTION
**Pull request by cTurtle98**
move helper functions to own file
change imports
change use of functions to reflect

### CODE REVIEW

File: src/seahawk_rov/seahawk_rov/helper_functions.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+cabrillorobotics@gmail.com
+'''
+
+# linear interpolation helper function
+def lerp(old_min:float, old_max:float, new_min:int, new_max:int, old_value:float):
+    '''linear interpolate helper function'''
+    old_range = old_max - old_min
+    new_range = new_max - new_min
+    new_value = (((old_value - old_min) * new_range) / old_range) + new_min
+    return new_value
```
Comment (OrionOth) [inline src/seahawk_rov/seahawk_rov/helper_functions.py]: Is it a good idea to force these types (`float`->`int`)? If so, I would suggest supplying the output as an int (`def lerp(...)->int:`) and casting `new_value` as such.

**Review by OrionOth — APPROVED** [review]
Looks good overall.

File: src/seahawk_rov/seahawk_rov/thrust_box_servo.py [hunk 2]
```diff
@@ -79,4 +55,4 @@ def __init__(self, node, i2c):
 
     def receive_thruster(self, message:Float32MultiArray):
         for thruster in self.thruster_map:
-            self.kit.servo[thruster].angle = int(lerp(-1.0, 1.0, 0, 3000, clamp(message.data[thruster], -1.0, 1.0)))
+            self.kit.servo[thruster].angle = int(seahawk_rov.lerp(-1.0, 1.0, 0, 3000, seahawk_rov.clamp(message.data[thruster], -1.0, 1.0)))
```
Comment (OrionOth) [inline src/seahawk_rov/seahawk_rov/thrust_box_servo.py]: Perhaps it would be a good idea to change `0` & `3000` to `PWM_MIN` and `PWM_MAX` constants

Comment (cTurtle98) [reply src/seahawk_rov/seahawk_rov/thrust_box_servo.py]: i un-genericized it for you

**Pull request merged**